contents/Documentation/Tutorials.playground/Pages/How to use nef library?.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,111 @@
+
+ In `nef` [version 0.4](https://github.com/bow-swift/nef/releases/tag/0.4.0), we have modularized the core so that you can use nef as in CLI as in your macOS projects. Taking advantage of the integration of SPM in Xcode, you can easily import `nef` in your project.
+ 
+ Just choose like package repository `https://github.com/bow-swift/nef.git` and Xcode will do the rest.
+ 
+ ![](/assets/nef-xcode-library.png)
+ 
+ Once Xcode solves the dependencies, you can import wherever you want, as a native framework:
+ ```swift
+ import nef
+ ```
+ That's all! The power of nef runs into your project. Cool!
+ */
+
+/*:
+ ## API
+ 
+ `nef` library provides several utilities to work with `Markdown`, `Jekyll` and `Carbon`. It enables easy access to render Markdown files, Jekyll content and Carbon images in a functional way.
+ 
+ ### Markdown
+ 
+If you need to render the content of a Playground page, or anything else that combine [**markup formatting**](https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_markup_formatting_ref/index.html) and code.
+ 
+ ```swift
+ nef.Markdown.render(content: String, toFile file: URL) -> IO<nef.Error, URL>
+ ```
+ 
+ ### Jekyll
+ 
+ When you need to render a content that combine Markdown, code, and [**nef commands**](https://github.com/bow-swift/nef#-generating-markdown-files-for-jekyll) with the Jekyll style.
+ 
+ ```swift
+ nef.Jekyll.render(content: String, toFile file: URL, permalink: String) -> IO<nef.Error, URL>
+ ```
+ 
+ ### Carbon
+ 
+ nef lets you render Carbon images given a CarbonModel; it is the configuration with the code and style.
+ 
+ ```swift
+ nef.Carbon.render(carbon: CarbonModel, toFile file: URL) -> IO<nef.Error, URL>
+ ```
+ 
+ 
+ You can find anothers helpers for drawing `Views` with a Carbon style.
+ 
+ ```swift
+ nef.Carbon.view(with configuration: CarbonModel) -> CarbonView
+ nef.Carbon.request(with configuration: CarbonModel) -> URLRequest
+ ```
+ 
+ */
+
+/*:
+ ## Usage
+ 
+ In this section, we will see how we can use the nef's API. As an example, we will take advantage of the `Carbon API`.
+
+ Firstly, we need to create a model with Carbon configuration and the style.
+ ```swift
+ let model = CarbonModel(code: """
+                               import nef
+                               let library = 'nef library is super cool!'
+                               """,
+                         style: CarbonStyle(background: .bow,
+                                            theme: .dracula,
+                                            size: .x1,
+                                            fontType: .firaCode,
+                                            lineNumbers: true, watermark: true))
+ ```
```

Comment:
This code could change in the future and we won't notice it. We can rely on local Cocoapods to import `nef` as a library on this playground and have this code compile-checked.

Reply:
Yep, I know about it, but as we do not have in available right now, I have let it for the next release, together with update to new playground system.